### ISSUE TITLE
Redo (2?) animations with US map 

### ISSUE THREAD

**Issue by dhowe** [dhowe/spectre#401]
![image](https://user-images.githubusercontent.com/737638/72606777-30653380-395a-11ea-8b4f-1720d4f79b34.png)

**Comment by billposters** [comment]
actioned with Stephen the animator, all work for £150!

**Comment by billposters** [comment]
Stephen has created the first versions of the US animations. @dhowe - you wanted to put this on hold for a bit whilst we discussed the content for the animation, can you update on your thoughts?

you can see the drafts in the drive here: https://drive.google.com/drive/folders/13Y6IkgkaFndhNjB40RY3vvC3KOt0gqO8

Note- the voter numbers need to be changed. 

qeuestion: 
- do we want to add the collegial vote numbers below the party logos?

**Comment by dhowe** [comment]
I don't remember what was on hold. I think we are just waiting on sizing info. But people in the US are quite familiar with these kind of election results animations, so we should match this expectation I guess (so yes to electoral college numbers)

**Comment by billposters** [comment]
OK sound good. I'll ask stephen to do a v2 of the animations including the college votes

**Comment by billposters** [comment]
3 x final Animations are in the drive : https://drive.google.com/drive/folders/1lDlfW3t1-fmgE3me6KHF9yVdVKOscfvK

**Comment by dhowe** [comment]
unless those have changed in the last few days they are already up on the production site (please check spectreknows and close this ticket if correct)
